services/processor/native/test/call_argument_test.go

```diff
@@ -78,11 +78,29 @@ func TestCallThatThrowsError(t *testing.T) {
 	assert.Equal(t, output.CallResult, protocol.EXECUTION_RESULT_ERROR_SMART_CONTRACT, "call result should be smart contract error")
 }
 
-func TestCallWithInvalidMissingErrorFails(t *testing.T) {
+func TestCallThatPanics(t *testing.T) {
+	h := newHarness()
+	call := processCallInput().WithMethod("BenchmarkContract", "panic").WithArgs().Build()
```

Comment:
Why are you calling `WithArgs()` passing nothing?

Reply:
It's a contract that receives zero arguments

Since it's a variadic function, I just call it empty. Do you want me to change it to `WithNoArgs()` to make it more clear?

Reply:
fixed